client/client_pool/src/concord_client_pool.cpp

```diff
@@ -42,9 +48,9 @@ SubmitResult ConcordClientPool::SendRequest(std::vector<uint8_t> &&request,
                                             uint64_t seq_num,
                                             std::string correlation_id,
                                             std::string span_context,
-                                            const bftEngine::RequestCallBack &callback) {
+                                            const SendCallback &callback) {
   if (callback && timeout_ms.count() == 0) {
-    callback(bftEngine::SendResult{SubmitResult::InvalidArgument});
+    callback(SendResult{SendError::InvalidArgument});
```

Comment:
It's a pity that we are not working in synch. We are changing the approach to the end-to-end error handling: https://confluence.eng.vmware.com/display/BLOC/BFT+client+error+handling+-+one+pager